cmake/onnxruntime_unittests.cmake

```diff
@@ -796,3 +796,17 @@ endif()
 list(APPEND onnxruntime_mlas_test_libs Threads::Threads)
 target_link_libraries(onnxruntime_mlas_test PRIVATE ${onnxruntime_mlas_test_libs})
 set_target_properties(onnxruntime_mlas_test PROPERTIES FOLDER "ONNXRuntimeTest")
+
+
+add_library(custom_op_library SHARED ${REPO_ROOT}/onnxruntime/test/testdata/custom_op_library/custom_op_library.cc)
+target_include_directories(custom_op_library PRIVATE ${REPO_ROOT}/include)
+if(UNIX)
+  if (APPLE)
```

Comment:
Should it also have a [link script](https://www.gnu.org/software/gnulib/manual/html_node/LD-Version-Scripts.html) for non-Win?

Reply:
leaving as is, since this is a test library anyway